packages/x-adapter-platform/src/endpoint-adapters/semantic-queries.endpoint-adapter.ts

```diff
@@ -0,0 +1,18 @@
+import { endpointAdapterFactory } from '@empathyco/x-adapter';
+import { SemanticQueriesResponse, SemanticQueriesRequest } from '@empathyco/x-types';
+import { semanticQueriesRequestMapper } from '../mappers/requests/semantic-queries-request.mapper';
+// eslint-disable-next-line max-len
+import { semanticQueriesResponseMapper } from '../mappers/responses/semantic-queries-response.mapper';
+
+export const semanticQueriesEndpointAdapter = endpointAdapterFactory<
+  SemanticQueriesRequest,
+  SemanticQueriesResponse
+>({
+  endpoint:
+    'https://api.{extraParams.env(.)}empathy.co/semantics-api/search-single/{extraParams.instance}',
```

Comment:
Maybe it's useful to make the search type dynamic like the instance? It can be `search-single` or `search-multiple`.

Anyways maybe right now doesn't make sense.